cli/src/main/kotlin/com/malinskiy/marathon/cli/ApplicationView.kt

```diff
@@ -59,7 +59,9 @@ fun main(args: Array<String>): Unit = mainBody(
             }
         } finally {
             stopKoin()
-            bugsnagExceptionsReporter.end()
+            if(bugsnagReporting){
```

Comment:
We're using the IntelliJ's automatic code style formatting, would you be able to hit the auto-format in the IDE? Alternatively, a space after `if` and after the closing bracket `)`.
Thank you.

Reply:
Sure, I just copied the existing approach from https://github.com/Malinskiy/marathon/blob/063a759ed9d416039936b95a71651fbe9abfa693/cli/src/main/kotlin/com/malinskiy/marathon/cli/ApplicationView.kt#L36, which I guess should be updated too, right?

Reply:
Yeah, seems fair. Thanks!